config/locales/client.en.yml

```diff
@@ -4212,10 +4212,10 @@ en:
       yes_delete_spammer: "Yes, Delete Spammer"
       ip_address_missing: "(N/A)"
       hidden_email_address: "(hidden)"
-      submit_tooltip: "Submit the private flag"
+      submit_tooltip: "Submit the flag"
       take_action_tooltip: "Reach the flag threshold immediately, hide the post, and agree with all pending flags"
       cant: "Sorry, you can't flag this post at this time."
-      notify_staff: "Notify staff privately"
```

Comment:
Again, I think we want to keep "privately" here unless you have a suggestion on a better formulation. Without "privately" it is not totally clear what happens with the flag and the user might think the flag will be public.